storm-client/src/jvm/org/apache/storm/Config.java

```diff
@@ -263,6 +263,13 @@
      */
     @IsPositiveNumber(includeZero = true)
     public static final String TOPOLOGY_METRICS_CONSUMER_CPU_PCORE_PERCENT = "topology.metrics.consumer.cpu.pcore.percent";
+
+    /**
+     * This config allows a topology to report metrics data points from the V2 metrics API through the metrics tick.
+     */
+    @IsBoolean
+    public static final String TOPOLOGY_ENABLE_V2_METRICS_TICK = "topology.enable.v2.metrics.tick";
```

Comment:
If this is enabled and `storm.metrics.reporters` is configured, will the metrics be sent to multiple places?

Reply:
Yes, v2 metrics would be reported by the V2 daemon reporter and the Topology v1 logging metrics consumer.